include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -147,6 +147,23 @@ constexpr auto align_pairwise(sequence_t && seq, alignment_config_t const & conf
     return align_pairwise(std::views::single(std::forward<sequence_t>(seq)), config);
 }
 
+//!\overload
+template <typename sequence_t, typename alignment_config_t>
+//!\cond
+    requires tuple_like<sequence_t> && !detail::align_pairwise_single_input<std::remove_reference_t<sequence_t>>
+//!\endcond
+constexpr auto align_pairwise(sequence_t && seq, alignment_config_t const & config)
```

Comment:
Ok, I thought about it, and this is how I'd like to tackle this.
Replace this definition: https://github.com/seqan/seqan3/blob/cbced425563b44fcd0a192762863d4f72d020d19/include/seqan3/alignment/pairwise/detail/concept.hpp#L109

with the following code:

```cpp
template <typename t>
SEQAN3_CONCEPT align_pairwise_single_input =
    sequence_pair<std::remove_reference_t<t>> &&
    std::is_lvalue_reference_v<t> ||
    (std::is_rvalue_reference_v<t> && 
     std::ranges::viewable_range<std::tuple_element_t<0, t>> && 
     std::ranges::viewable_range<std::tuple_element_t<1, t>>);
```
Then replace this line: https://github.com/seqan/seqan3/blob/cbced425563b44fcd0a192762863d4f72d020d19/include/seqan3/alignment/pairwise/align_pairwise.hpp#L134

with this: 
```cpp
    requires detail::align_pairwise_single_input<sequence_t> &&
```

Reply:
Then we don't need the this additional interface.